packages/volto/theme/themes/pastanaga/collections/form.overrides

```diff
@@ -123,16 +123,50 @@
   z-index: 1;
   top: 0;
   right: 0;
+  left: auto;
   display: flex;
   height: 60px;
   align-items: center;
   margin-right: 1rem;
+  background: transparent;
+  box-shadow: none;
+  transform: none;
 
   .item {
     cursor: pointer;
   }
 }
 
+.block.form {
```

Comment:
@sneridagh @robgietema This pull requests mentions that it improves the css for the SchemaWidget
I see that it is assigned to ContentTypeSchema as such it should affect pages such as
https://volto.demo.plone.org/controlpanel/dexterity-types/Document/schema
However I don't see any .block.form.

Please give guidance on how to find the page or component that this css touches.

Reply:
@ichim-david this will be useful in the case that the widget lives inside a block. Which is the next step that we are looking forward to.

![image (1)](https://github.com/user-attachments/assets/a79267a8-bad9-4abf-b685-c03d0340ca4c)

Reply:
I am not a fan of the important on the segment > div backend and neither of the static height: 66px but I also don't have any content that would be affected by this style.
Perhaps the folks from Redturtle with products such as volto-form-block might be affected.
if that's the case then @pnicolli have a look at this css as well

Reply:
@ichim-david I guess that's because we have to overcome other of semanticUI definitions that already have an `!important`... @robgietema did these the other day, so I guess that's why. As you said, since they are for that use case, and it's touching semanticUI, I'd not bother much.